app/views/standard_layout.scala.html

```diff
@@ -109,7 +109,7 @@
                             <li><a href="@uk.gov.hmrc.cataloguefrontend.shuttering.routes.ShutterEventsController.shutterEvents">Shutter Events</a></li>
                             <li role="separator" class="divider"></li>
                             <li><a href="@HealthIndicatorsController.indicatorsForRepoType()">Health-Indicators</a></li>
-                            <li><a href="@uk.gov.hmrc.cataloguefrontend.platforminitiatives.routes.PlatformInitiativesController.platformInitiatives(DisplayType.Chart)">Platform Initiatives</a></li>
+                            <li><a href="@uk.gov.hmrc.cataloguefrontend.platforminitiatives.routes.PlatformInitiativesController.platformInitiatives("", DisplayType.Chart)">Platform Initiatives</a></li>
```

Comment:
we should be able to move the default values into app.routes

Reply:
Removed unnecessary default values